<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BDC8-LX-5111KRH" name="Regression_Utilities.HPETest" tests="1" failures="0" timestamp="18 May 2017 18:38:12 GMT" time="57.201" errors="1">
  <testcase name="TC_134_ValidateErrorMessageForWrongUsername" time="57.201" classname="Regression_Utilities.HPETest">
    <error type="org.openqa.selenium.NoSuchElementException" message="Element with by xpath &#039;xpath|(//*[@label=&quot;Address&quot;])[2]			&#039; wasn&#039;t found.
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;2.48.2&#039;, revision: &#039;41bccdd10cf2c0560f637404c2d96164b67d9d67&#039;, time: &#039;2015-10-09 13:08:06&#039;
System info: host: &#039;BDC8-LX-5111KRH&#039;, ip: &#039;192.168.1.5&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_121&#039;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Element with by xpath 'xpath|(//*[@label="Address"])[2]			' wasn't found.
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'BDC8-LX-5111KRH', ip: '192.168.1.5', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: driver.version: unknown
	at com.perfectomobile.selenium.MobileApplication.getSingleElement(MobileApplication.java:369)
	at com.perfectomobile.selenium.MobileApplication.findElementByXPath(MobileApplication.java:211)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at com.perfectomobile.selenium.MobileApplication.findElement(MobileApplication.java:150)
	at Libraries.HPEOperations.EnterURL(HPEOperations.java:221)
	at Libraries.HPEOperations.Install_App_HPE_GO_iphone7and6S(HPEOperations.java:333)
	at Libraries.HPEOperations.Install_App_HPE_GO_iOS(HPEOperations.java:367)
	at Libraries.HPECommonFunctions.Open_App_HPE_iOS(HPECommonFunctions.java:148)
	at Libraries.HPECommonFunctions.Open_App_HPE(HPECommonFunctions.java:126)
	at Regression_Utilities.HPETest.TC_134_ValidateErrorMessageForWrongUsername(HPETest.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:391)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- TC_134_ValidateErrorMessageForWrongUsername -->
</testsuite> <!-- Regression_Utilities.HPETest -->
